compute/cloud-client/src/main/java/compute/disks/CreateDiskSecondaryCustom.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute.disks;
+
+//[START compute_disk_create_secondary_custom]
+import com.google.cloud.compute.v1.Disk;
+import com.google.cloud.compute.v1.DiskAsyncReplication;
+import com.google.cloud.compute.v1.DisksClient;
+import com.google.cloud.compute.v1.GuestOsFeature;
+import com.google.cloud.compute.v1.Operation;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CreateDiskSecondaryCustom {
```

Comment:
Considering refactoring the class name to `CreateSecondaryCustomDisk`.

Reply:
Thank you! Renamed